app/controllers/proposed_organisations_controller.rb

```diff
@@ -13,8 +13,11 @@ def update
     end
     proposed_org = ProposedOrganisation.find params[:id]
     if update_param == "accept"
+      org = proposed_org.accept_proposal
       flash[:notice] = "You have approved the following organisation"
-      redirect_to organisation_path(proposed_org.accept_proposal) and return false
+      result_of_inviting = InviteUnregisteredUserFromProposedOrg.new(org.email,org).run
```

Comment:
these conditional branches are getting a little long - could we break each of them out into separate methods?